src/components/Footer/__tests__/Footer.test.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Footer } from '../index';
+
+describe('Footer', () => {
```

Comment:
We should also add a snapshot test, since this shouldn't change very often and doesn't have much logic.